core/services/addon/api/IncompatibleAddonHandler.php

```diff
@@ -28,7 +28,7 @@ public function deactivateIncompatibleAddons()
         $this->deactivateIncompatibleAddon(
             'WP Users Integration',
             'EE_WPUSERS_VERSION',
-            '2.1.1.rc.001',
+            '2.1.0.rc.003',
```

Comment:
Can this not be set to the current release version?

And... having said that, if the current release version works with the legacy editor as is, do we need to set this as incompatible?